tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38, py39, flake8, filename_matching, mypy, individual_coverage, docs
+envlist = py, flake8, mypy
```

Comment:
this list only affects what is run into actively for us developers?
Or does this also affect what is run in CI?

Reply:
My understanding is that this only affects what is run when you just type `tox` on the command line, see the commit message.